sys/net/gnrc/network_layer/sixlowpan/frag/rb/gnrc_sixlowpan_frag_rb.c

```diff
@@ -138,6 +141,47 @@ gnrc_sixlowpan_frag_rb_t *gnrc_sixlowpan_frag_rb_add(gnrc_netif_hdr_t *netif_hdr
     return (res < 0) ? NULL : &rbuf[res];
 }
 
+
+
+bool gnrc_sixlowpan_frag_rb_exists(const gnrc_netif_hdr_t *netif_hdr,
+                                   uint16_t tag)
+{
+    return (_rbuf_get_by_tag(netif_hdr, tag) != NULL);
+}
+
+void gnrc_sixlowpan_frag_rb_rm_by_datagram(const gnrc_netif_hdr_t *netif_hdr,
+                                           uint16_t tag)
+{
+    gnrc_sixlowpan_frag_rb_t *e = _rbuf_get_by_tag(netif_hdr, tag);
+
+    if (e != NULL) {
+        gnrc_sixlowpan_frag_rb_remove(e);
+    }
+}
+
+static gnrc_sixlowpan_frag_rb_t *_rbuf_get_by_tag(const gnrc_netif_hdr_t *netif_hdr,
+                                                  uint16_t tag)
+{
+    const uint8_t *src = gnrc_netif_hdr_get_src_addr(netif_hdr);
```

Comment:
`assert(netif_hdr != NULL);` should go before that statement.

Reply:
Yepp, I'm on that. ;-)

Reply:
Just squash directly.

Reply:
Fixed.